google-cloud-tools-plugin/src/com/google/gct/idea/ui/GoogleCloudToolsIcons.java

```diff
@@ -31,7 +35,7 @@
   public static final Icon CLOUD= load("/icons/cloudPlatform.png");
   public static final Icon CLOUD_MODULE = load("/icons/google_cloud_module.png");
   public static final Icon REFRESH = load("/icons/refresh.png");
-  public static final Icon[] STEP_ICONS = findStepIcons("/icons/step_");
+  public static final List<Icon> STEP_ICONS = findStepIcons("/icons/step_");
```

Comment:
Since this is part of the public API, can we make it an explicit c.g.common.collect.ImmutableList? I hate things that look like they're muable and aren't.